doc/modules/model_evaluation.rst

```diff
@@ -1109,6 +1110,53 @@ function:
     >>> matthews_corrcoef(y_true, y_pred)  # doctest: +ELLIPSIS
     -0.33...
 
+.. _multilabel_confusion_matrix:
+
+Multi-label confusion matrix
+----------------
+
+The :func:`multilabel_confusion_matrix` function compute multi-label confusion
+matrix label-wisely (default) or sample-wisely (samplewise=True) to evaluate
+the accuracy of a classification.
```

Comment:
Add that: multilabel_confusion_matrix also treats multiclass data is also treated as if it were multilabel, as this is a transformation commonly applied to evaluate multiclass problems with binary classification metrics (such as precision, recall, etc.).